\documentclass[10pt]{article}
\usepackage{graphicx}
\usepackage{calc}
\usepackage{comment}
\usepackage{soul}

\makeatletter
\newlength{\bibhang}
\setlength{\bibhang}{1em} %1em}
\newlength{\bibsep}
{\@listi \global\bibsep\itemsep \global\advance\bibsep by\parsep}
\newenvironment{bibsection}%
{\begin{enumerate}{}{%
			\setlength{\leftmargin}{\bibhang}%
			\setlength{\itemindent}{-\leftmargin}%
			\setlength{\itemsep}{\bibsep}%
			\setlength{\parsep}{\z@}%
			\setlength{\partopsep}{0pt}%
			\setlength{\topsep}{0pt}}}
	{\end{enumerate}\vspace{-.6\baselineskip}}
\makeatother

\reversemarginpar

\usepackage{titlesec}

\usepackage[paper=letterpaper,
%includefoot, % Uncomment to put page number above margin
marginparwidth=1.2in,     % Length of section titles
marginparsep=.05in,       % Space between titles and text
margin=1in,               % 1 inch margins
includemp]{geometry}

\setlength{\parindent}{0in}

\usepackage[shortlabels]{enumitem}

\usepackage{fancyhdr,lastpage}
\pagestyle{fancy}
\fancyhf{}\renewcommand{\headrulewidth}{0pt}
\fancyfootoffset{\marginparsep+\marginparwidth}
\newlength{\footpageshift}
\setlength{\footpageshift}
{0.5\textwidth+0.5\marginparsep+0.5\marginparwidth-2in}
\lfoot{\hspace{\footpageshift}%
	\parbox{4in}{\, \hfill %
		\arabic{page} of \protect\pageref*{LastPage} % +LP
		\hfill \,}}

\usepackage{color,hyperref}
\definecolor{darkblue}{rgb}{0.0,0.0,0.3}
\hypersetup{colorlinks,breaklinks,
	linkcolor=darkblue,urlcolor=darkblue,
	anchorcolor=darkblue,citecolor=darkblue}



\newcommand{\makeheading}[2][]%
{\hspace*{-\marginparsep minus \marginparwidth}%
\begin{minipage}[t]{\textwidth+\marginparwidth+\marginparsep}%
		{\large \bfseries #2 \hfill #1}\\[-0.15\baselineskip]%
		\rule{\columnwidth}{1pt}%
\end{minipage}}

\renewcommand{\section}[1]{\pagebreak[3]%
	\hyphenpenalty=10000%
	\vspace{1.3\baselineskip}%
	\phantomsection\addcontentsline{toc}{section}{#1}%
	\noindent\llap{\scshape\smash{\parbox[t]{\marginparwidth}{\raggedright\textcolor{black}{#1}}}}%
	\vspace{-\baselineskip}\par}

\newenvironment{outerlist}[1][\enskip\textbullet]%
{\begin{itemize}[#1,leftmargin=*]}{\end{itemize}%
	\vspace{-.6\baselineskip}}

\newenvironment{lonelist}[1][\enskip\textbullet]%
{\begin{list}{#1}{%
			\setlength{\partopsep}{0pt}%
			\setlength{\topsep}{0pt}}}
	{\end{list}\vspace{-.6\baselineskip}}

\newenvironment{innerlist}[1][\enskip\textbullet]%
{\begin{itemize}[#1,leftmargin=*,parsep=0pt,itemsep=0pt,topsep=0pt,partopsep=0pt]}
	{\end{itemize}}

\newenvironment{loneinnerlist}[1][\enskip\textbullet]%
{\begin{itemize}[#1,leftmargin=*,parsep=0pt,itemsep=0pt,topsep=0pt,partopsep=0pt]}
	{\end{itemize}\vspace{-.6\baselineskip}}

\newcommand{\blankline}{\quad\pagebreak[3]}
\newcommand{\halfblankline}{\quad\vspace{-0.5\baselineskip}\pagebreak[3]}

\newcommand\doilink[1]{\href{http://dx.doi.org/#1}{#1}}
\newcommand\doi[1]{doi:\doilink{#1}}

\providecommand*\url[1]{\href{#1}{#1}}
\renewcommand*\url[1]{\href{#1}{\texttt{#1}}}

\providecommand*\email[1]{\href{mailto:#1}{#1}}

\providecommand\BibTeX{{B\kern-.05em{\sc i\kern-.025em b}\kern-.08em
		\TeX}}
\providecommand\Matlab{\textsc{Matlab}}

\begin{document}
	\makeheading{NELSON KIPCHIRCHIR KIBINGE}


%\makeheading[\includegraphics[height=1.5in]{photo}]{Your Name}%
\vspace{.1in}
\section{Contact Information}
% NOTE: Mind where the & separators and \\ breaks are in the following
%       table.
%
% ALSO: \rcollength is the width of the right column of the table
%       (adjust it to your liking; default is 1.85in).
%
\newlength{\rcollength}\setlength{\rcollength}{1.4in}%
%
\begin{tabular}[t]{@{}p{\textwidth-\rcollength}p{\rcollength}}
%\href{http://www.cse.osu.edu/}%
%     {Department of Computer Science and Engineering} & \\
%\href{http://www.osu.edu/}{The Ohio State University}
Second Floor  \\
R n D building Orionintie 1\\
Digital Medicine, Orion Pharma\\ 
Tel: +358509663906\\
\email{nelson.kibinge@orionpharma.com}\\
\end{tabular}

%\section{Objective}

%Insert text here if you want to
%\begin{innerlist}
%\item More information and auxiliary documents can be found at\\\url{http://www.tedpavlic.com/facjobsearch/}
%\end{innerlist}


%\section{Personal Profile}
%\begin{tabular}[t]{@{}p{\rcollength}p{\rcollength}}
%Age: & 32 years \\
%Status: & Married\\
%Nationality: & Kenya\\
%Languages: & English(Fluent) Japanese(Conversational) and Kiswahili(Native), 
%\end{tabular}
\section{Professional summary}

A  computational biologist interested in data-driven and cutting edge tool development for applications in drug discovery and academic research. I possess extensive experience and set of skills working with multiple teams and in lead roles. My major interest is in the production of applied statistics and software code utilized in clinical trials, epidemiological studies and therapeutic target discovery. As a PhD-trained scientist, I am involved in diverse research projects and in teaching postgraduate courses. 


\section{Core competencies}%

\begin{innerlist}

	\item Drug-Target and Biomarker discovery
	\item Statistical Genomics
	\item Clinical trial statistics
	\item Protein mass spectrometry
	\item protein microarrays
	\item Systems Biology
	\item Machine Learning
	\item Next generation sequencing (MiSeq and Oxford Nanopore)
	\item Compartmental Epidemiological Modeling
	\item Advanced R programming
	\item Intermediate level perl and basic python programming
	\item Unix shell scripting,
	\item Tensorflow

	
	
\end{innerlist}

\halfblankline%



\section{Professional Experience}

\textbf{Senior Scientist} \hfill {Feb 2020 to present}
\begin{innerlist}
	\item[] Computational Biology, Digital Medicine Department\\
	Orion Pharma\\
\end{innerlist}



\section{Professional Experience}

\textbf{Senior Research Associate} \hfill {June 2019 to present}
\begin{innerlist}
	\item[] Genetic Epidemiology Lab, MRC IEU\\
	University of Bristol\\
\end{innerlist}


\textbf{Senior Postdoctoral Scientist}  \hfill {Nov 2018 to present}
\begin{innerlist}
	\item[] Senior postdoctoral scientist leading the biostatistics department \\
	KEMRI-Wellcome Trust Research Programe, Kenya\\
\end{innerlist}



\textbf{Visiting Research Fellow} \hfill {Feb 2019 to Mar 2019}
\begin{innerlist}
	\item[] Project: \emph{ Multi-center malaria antigen discovery project}\\
	University of Heidelberg Germany\\
\end{innerlist}


\textbf{Visiting Research Fellow (Mathematical modeling)} \hfill {Feb 2017 to May 2017}
\begin{innerlist}
	\item[] Project: \emph{ Modeling the genetic diversification in RSV transmission }\\
	University of Warwick, United Kingdom\\
\end{innerlist}

\newpage

\textbf{Early Career Postdoctoral Research Fellow} \hfill {Apr 2016 to Dec 2018}
\begin{innerlist}
	
	\item[] Project: \emph{Evaluating local respiratory syncytial virus (RSV) genetic diversity: a statistical perspective}\\
	Virus Epidemiology and Control Group,\\
	KEMRI-Wellcome Trust Research Programe, Kenya\\
\end{innerlist}
%\newpage
\textbf{Research Assistant} \hfill {Apr 2013 to March 2016}
\begin{innerlist}
	
	\item[] Project: \emph{Data mining methods for understanding gene expression in high througphut technologies}\\
	Computational systems biology laboratory,\\
	Nara Institute of Science and Technology\\
	Supervisor: Kanaya Shigehiko, Ph.D
\end{innerlist}

\halfblankline

\textbf{Research Assistant} \hfill {Jun 2007 to May 2008}
\begin{innerlist}
	
	\item[] Project: \emph{Biosafety; gene flow in wild oryzae sativa in Mwea region}\\
	School of Biological Sciences,\\
	University of Nairobi\\
	Supervisor: Samuel Kiboi, Ph.D
\end{innerlist}

\section{Teaching Experience}
\textbf{Part-time Lecturer} \hfill {September 2017 to Present}
\begin{innerlist}
	\item[] MSc Course: Biostatistics and Research Methods\\
	Courses taught: Biostatistics, databases and data visualization,\\
	Research methods and experimental design\\
	Department of Biochemistry,\\
	Pwani University, Kenya
\end{innerlist}

\halfblankline

\textbf{Teaching Assistant} \hfill {Dec 2012 to Mar 2013}
\begin{innerlist}
	\item[] Bioinformatics II\\
	Professor: Kensuke Nakamura\\
	Department of Life Science and Informatics,\\
	Maebashi Institute of Technology, Japan
\end{innerlist}

\halfblankline

\textbf{Teaching Assistant} \hfill {Oct 2011 to Oct 2014}
\begin{innerlist}
	\item[] Systems Biology\\
	Professor: Md Altaf-Ul-Amin\\
	Graduate School of Information Science,\\
	Nara Institute of Science and Technology, Japan
\end{innerlist}

\section{Other Work Experience}
\textbf{Medical Representative} \hfill{Sep 2008 to Dec 2009}
\begin{innerlist}
\item[] Pharmaken Limited Kenya\\
        Nairobi office\\
\end{innerlist}


\section{Education}%

\href{http://www.naist.jp/en/}{\textbf{Nara Institute of Science and Technology (NAIST)}},Japan
\begin{outerlist}
	\item[] Ph.D. {Bioinformatics},
	March 2016
	\begin{innerlist}
		\item Thesis title: \emph{Transcription regulation network analysis towards integrative big data biology}
		\item Advisors:
		%\href{http://www.biostat.umn.edu/~brad/}
		{Shigehiko KANAYA, PhD} and
		%\href{http://www.biostat.umn.edu/~sudiptob/}
		{Naoaki ONO, PhD}
	\end{innerlist}
	
	\item[] M.Sc. {Bioinformatics and biostatistics}, March 2013
	\begin{innerlist}
		%\item Thesis Topic: \emph{Plasmodium phylogenetics and numeric modification of amino acid residues }
		\item Advisors:
		%\href{http://www.biostat.umn.edu/~brad/}
		{Shigehiko KANAYA, PhD} and {Kensuke NAKAMURA,PhD}
	\end{innerlist}
\end{outerlist}

\vspace{.1in}
\href{http://www.uonbi.ac.ke/}{\textbf{University of Nairobi}},
Nairobi Kenya

\begin{outerlist}
	\item[] B.Sc. {Biology} (First Class Honours), October 2008
	\begin{innerlist}
		\item Advisor: Samuel Kiboi, Ph.D
	\end{innerlist}
	
\end{outerlist}


\section{Funding and Awards}%

Funding
\begin{innerlist}
	\item IDeAL postdoctoral funding  \hfill {Apr 2016 to Sep 2018} \hfill {GBP: 175000}
	\item TDModNet tropical disease modeling seed fund \hfill {Jan 2017 to Jun 2017} \hfill {GBP: 4600}
	\item KEMRI Institutional Research Grant (IRG) \hfill {June 2018 to Dec 2019} \hfill {KES: 3967814}
	
\end{innerlist}%
\halfblankline%


Awards

\begin{innerlist}
	\item Japanese Language studentship Osaka University \hfill {Apr 2010 to Sep 2010}
	\item Research Fellowship   \hfill {Oct 2010 to Mar 2011}
	\item Masters Scholarship  \hfill {Apr 2011 to Mar 2013}
	\item Doctoral Course Scholarship  \hfill {Apr 2013 to Mar 2016}
	\item Gandhi Smarak Nidhi Trustee Fund Prize (Top undergraduate student SBS) \hfill May 2011
\end{innerlist}%

\halfblankline%

\section{Professional Membership}%
Scientific Societies \hfill {May 2010 -- Present}
\begin{innerlist}%
	\item Molecular Biology Society of Japan (MBSJ)
	\item International Society of Computational Biology (ISCB)
	\item African Society for Bioinformatics and Computational Biology (ASBCB)
\end{innerlist}%

\halfblankline%

Student Societies \hfill {July 2004 -- Aug 2008}
\begin{innerlist}
	\item Biology Association of Nairobi University Students(BANUS) \hfill {July 2004 to July 2007}
	\item Organising Secretary BANUS \hfill {Aug 2007 to July 2008}
\end{innerlist}

\halfblankline




\section{Refereed Journal Publications}
\vspace{-.1275in}
\begin{bibsection}
	
	\item \textbf{Kibinge, N. K}., Relton, C. L., Gaunt, T. R., and Richardson, T. G. (2020). Characterizing the Causal Pathway for Genetic Variants Associated with Neurological Phenotypes Using Human Brain-Derived Proteome Data. The American Journal of Human Genetics, 106(6), 885-892.
	
	\item Kamuyu, G., Tuju, J., Kimathi, R., Mwai, K., Mburu, J., \textbf{Kibinge, N.}, ... and Njunge, J. M. (2018). KILchip v1. 0: a novel Plasmodium falciparum merozoite protein microarray to facilitate malaria vaccine candidate prioritization. Frontiers in immunology, 9.
	
	\item Githinji, G., Agoti, C. \textbf{N., Kibinge, N.}, Henson, S., Munywoki, P., Brand, S., ... and Worby, C. J. (2018). Assessing the utility of minority variant composition in elucidating RSV transmission pathways. bioRxiv, 411512.
	
	\item Mohammed, K. S., \textbf{Kibinge, N.}, Prins, P., Agoti, C. N., Cotten, M., Nokes, D. J., and Githinji, G. (2018). Evaluating the performance of tools used to call minority variants from whole genome short-read data. \textit{Wellcome Open Research, 3.}
	
	\item Njunge, J. M., Oyaro, I. N., \textbf{Kibinge, N. K.}, Rono, M. K., Kariuki, S. M., Newton, C. R.,  and Gitau, E. N. (2017). Cerebrospinal fluid markers to distinguish bacterial meningitis from cerebral malaria in children. \textit{Wellcome Open Research, 2}.
	
	\item \textbf{Kibinge, N.}, Ono, N., Horie, M., Sato, T., Sugiura, T., Altaf-Ul-Amin, M. and Kanaya, S. (2016). Integrated pathway-based transcription regulation network mining and visualization based on gene expression profiles. \textit{Journal of biomedical informatics}, 61, 194-202. 
	
	\item \textbf{Kibinge, N}., and Kiboi, S. (2016). Increasing the role of systems biology towards leveraging ecology research in Kenya. \textit{CICSJ Bulletin Japan}, 34(2), 59.
	
	\item Tanaka, N., Zakaria, N. A., \textbf{Kibinge, N.} K., Kanaya, S., Tamura, T., and Yoshida, M. (2014). Fall-risk classification of the timed up-and-go test with principal component analysis. \textit{International journal of neurorehabilitation}, 1, e106-e106.
	
	\item \textbf{Kibinge, N.}, Ikeda, S., Ono, N., Altaf-Ul-Amin, M., and Kanaya, S. (2014). Integration of Residue Attributes for Sequence Diversity Characterization of Terpenoid Enzymes. \textit{BioMed Research International}, 2014.
	\item Ohtana, Y., Abdullah, A. A., Altaf‐Ul‐Amin, M., Huang, M., Ono, N., Sato, T.and \textbf{Kibinge, N. K}. (2014). Clustering of 3D Structure Similarity Based Network of Secondary Metabolites Reveals Their Relationships with Biological Activities. \textit{Molecular Informatics}, 33(11‐12), 790-801
	\item Afendi, F. M., Ono, N., Nakamura, Y., Nakamura, K., Darusman, L. K., \textbf{Kibinge, N.}, and Kanaya, S. (2013). Data mining methods for omics and knowledge of crude medicinal plants toward big data biology. \textit{Computational and structural biotechnology journal}, 4.
	\item Ikeda, S., Abe, T., Nakamura, Y.,\textbf{ Kibinge, N.}, Morita, A. H., Nakatani, A., ... and Kanaya, S. (2013). Systematization of the protein sequence diversity in enzymes related to secondary metabolic pathways in plants, in the context of big data biology inspired by the KNApSAcK Motorcycle database. \textit{Plant and Cell Physiology}, pct041.
	
\end{bibsection}




\section{Manuscripts in Preparation/ Submitted}
\vspace{-.1275in}
\begin{bibsection}
	
	\item \textbf{Nelson Kibinge}, Charles Sande, Ken Mwai, George Githinji, Abdirahman Abdi, James Nokes and James Njunge, TransProt: An analysis pipeline for proteomic and transcriptomic data. 
	\item \textbf{Nelson Kibinge}, Samuel Brand, George Githinji, Charles Agoti, Lister Mirarwa, Everlyn Kamau, James Otieno and James Nokes, Sampling coverage and variant diversity capture in RSV surveillance  \textit{Manuscript in prep}. 
	
	
\end{bibsection}




\section{Conference and Poster Presentations}%
\vspace{-.1in}

\begin{bibsection}
	

\item \textbf{Nelson Kibinge}, Samuel Brand, George Githinji, Charles Agoti, James Otieno, Moses Kiti, Matthew Keeling and James Nokes (2017), Sampling considerations for optimizing genetic diversity capture in Respiratory Syncytial virus Surveilance \textit{Epidemics 6 2017}, SItges, Spain, November 28-December 2, 2017
	
	
	\item \textbf{Nelson Kibinge}, Naoaki Ono, Masafumi Horie, Ming Huang, Tetsuo Sato, Tadao Sugiura, Md. Altaf-Ul-Amin, Saito Akira and Shigehiko Kanaya (2014), A systems mapping of transcription regulation in genes and modules of genes in lung cancer pathways, \textit{GIW-ISCB 2014}, Odaiba, Tokyo, December 14-17, 2014.
	
	\item \textbf{Nelson Kibinge}, Shun Ikeda, Naoaki Ono, Nakamura Kensuke, MD Altaf-Ul-Amin and Shigehiko Kanaya (2012). Utilizing metric properties of amino acids in distance matrix methods for protein sequence analysis BMB2012 \textit{The 35th annual meeting of the moleculars biology society of Japan Fukuoka}, December, 11–14, 2012
	
	\item \textbf{Nelson Kibinge}, Shun Ikeda, Farit Afendi, Naoaki Ono, Nakamura Kensuke, MD Altaf-Ul-Amin and Shigehiko Kanaya (2012). Phylogenetic analyses suggest influence of parasitism on the genomes of malaria parasite. \textit{Genomics of Common Diseases 2012}, Potomac MD, September 19-22, 2012
	
	\item Shun Ikeda, Manabu Ohi, Atsushi Nakatani, \textbf{Nelson Kibinge}, Naoaki Ono, Md. Altaf-UI-Amin, Shigehiko Kanaya (2012). Codon usage patterns of pathogenic bacteria. \textit{Genomics of Common Diseases 2012, Potomac MD}, September 19-22, 2012
\end{bibsection}%



\newpage
\section{Referees}



Professor Philip Bejon
\begin{innerlist}
\item [] Executive Director\\
Center for Geographic Medical Research\\
KEMRI-Wellcome Trust Kilifi\\
P.O Box 230-80108 Kilifi\\
%Tel: +254722641204\\
E-mail: \href{mailto:pbejon@kemri-wellcome.org}{pbejon@kemri-wellcome.org}
\end{innerlist}

\halfblankline%

 Professor Suhaila Hashim
 \begin{innerlist}
 	\item[]  Head of Department\\
 	Department of Biochemistry\\
 	Center for Geographic Medical Research\\
 	Pwani University, Kenya\\
 	P.O Box 195-80108 Kilifi\\
 	Tel: +254707719757\\
 	E-mail: \href{mailto:s.hashim@pu.ac.ke}{s.hashim@pu.ac.ke}
 \end{innerlist}
 
\halfblankline

Dr. Samuel Kiboi
\begin{innerlist}
	\item[]  Senior Lecturer\\
	School of Biological Sciences\\
	University of Nairobi\\
	P.O BOX 30197-00100\\
	Nairobi\\
	Tel: +254 724580929\\
	E-mail: \href{mailto:samuel.kiboi@uonbi.ac.ke}{samuel.kiboi@uonbi.ac.ke}
	
\end{innerlist}

 \halfblankline



Professor Shigehiko Kanaya
\begin{innerlist}
\item[]  Professor\\
Graduate School of Information Science\\
Nara Institute of Science and Technology\\
8916-5 Takayama Ikoma\\
Nara 630-0101\\
Tel: +81 743 72 5952\\
E-mail: \href{mailto:skanaya@gtc.naist.jp}{skanaya@gtc.naist.jp}

\end{innerlist}



\halfblankline%
\begin{comment}
 Kensuke Nakamura, Ph.D
\begin{innerlist}
\item [] Professor\\
Department of Life Science and Informatics\\
Maebashi Institure of Technology\\
460-1 Kamisadori-cho, Maebashi\\
371-0816 Japan\\
TEL: +81 27 265 7324\\
E-mail: \href{mailto:knakamura@maebashi-it.ac.jp}{knakamura@maebashi-it.ac.jp}
\end{innerlist}



Md Altaf-Ul-Amin
\begin{innerlist}
\item[]  Associate Professor \hfill {Phone: PHONE NUMBER}\\
Division of Bioinformatics and Genomics \hfill{E-mail: EMAIL ADRESS}\\
Nara Institute of Science and Technology
\end{innerlist}


\halfblankline

Naoaki Ono
\begin{innerlist}
\item[]  Assistant Professor \hfill {Phone: PHONE NUMBER}\\
Division of Bioinformatics and Genomics \hfill{E-mail: EMAIL ADRESS}\\
Nara Institute of Science and Technology
\end{innerlist}
\end{comment}

\end{document}

%%%%%%%%%%%%%%%%%%%%%%%%%% End CV Document %%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------%
% The following is copyright and licensing information for
% redistribution of this LaTeX source code; it also includes a liability
% statement. If this source code is not being redistributed to others,
% it may be omitted. It has no effect on the function of the above code.
%----------------------------------------------------------------------%
% Copyright (c) 2007, 2008, 2009, 2010, 2011 by Theodore P. Pavlic
%
% Unless otherwise expressly stated, this work is licensed under the
% Creative Commons Attribution-Noncommercial 3.0 United States License. To
% view a copy of this license, visit
% http://creativecommons.org/licenses/by-nc/3.0/us/ or send a letter to
% Creative Commons, 171 Second Street, Suite 300, San Francisco,
% California, 94105, USA.
%
% THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
% OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
% MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
% IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
% CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
% TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
% SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
%----------------------------------------------------------------------%
